common/SmokeTests/Samples/IotHubConnection/IotHubConnection.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
```

Comment:
Using a dedicated project file here is intended to allow adjusting the package references for use with the nightly feeds without impacting the actual sample which is bound to the GA versions.

Reply:
Is the purpose of doing this partially to reduce the chances of the smoke test package version updater mutating the csproj in the samples folder and having that get accidentally checked in?